ui/initial_state.py

```diff
@@ -75,6 +83,15 @@ def to_state_file(book_path):
             current_book = main_state['current_book']
             if not current_book == manual_filename:
                 current_book = os.path.join(path, current_book)
+        if 'current_language' in main.state:
```

Comment:
That's an error, should be `main_state`.

```
  File "/home/kaspar/projects/braille/code/canute-ui/ui/initial_state.py", line 86, in read_user_state
    if 'current_language' in main.state:
NameError: name 'main' is not defined
```

 Would save me some time in reviewing if I could you rely on pushed code being in a working state. If you fix this and the lint errors then Travis status would be green and can help you spot oversights like this if you aim to keep it green.